src/dataframe/dataframe.jl

```diff
@@ -589,12 +580,14 @@ end
 ##############################################################################
 
 """
-    insertcols!(df::DataFrame, ind::Int; name=col,
-                makeunique::Bool=false)
-    insertcols!(df::DataFrame, ind::Int, (:name => col)::Pair{Symbol,<:AbstractVector};
-                makeunique::Bool=false)
+    insertcols!(df::DataFrame, ind::Int, (name=>col)::Pair{Symbol,<:AbstractVector}...;
+                makeunique::Bool=false, copycols::Bool=true)
+    insertcols!(df::DataFrame, (name=>col)::Pair{Symbol,<:AbstractVector}...;
+                makeunique::Bool=false, copycols::Bool=true)
 
 Insert a column into a data frame in place. Return the updated `DataFrame`.
+If `ind` argument is omitted it is by default set to `ncol(df)+1`
+(the column is inserted as a last column of a `DataFrame`).
 
 # Arguments
 - `df` : the DataFrame to which we want to add a column
```

Comment:
```suggestion
- `df` : the DataFrame to which we want to add columns
```
Same below for other arguments.